src/app/registration/contact-form/contact-form.component.html

```diff
@@ -103,25 +103,36 @@ <h3>
           <label class="mb-2" for="phone">
             Phone Number:
           </label>
-          <input 
-            class="form-control mb-1" 
-            type="tel" 
-            formControlName="phone" 
-            id="phone" 
-            name="phone" 
-            placeholder="111-222-3333"
-            maxlength="12"
-            appPhoneFormat 
-            />
-          <div *ngIf="myForm.errors?.['phoneRequired']">
-            <small *ngIf="myForm.errors?.['phoneRequired']" class="text-danger">*Your Phone Number is Required</small>
+        <ngx-intl-tel-input
+            #phoneInput
+            [cssClass]="'custom'"
```

Comment:
Add the `form-control` class to match the field style to other fields on the page.

Reply:
I added some custom css to match the styling of the other fields. Adding the form-control class was conflicting with the styles provided from the intl-tel-input library.

Addressed in: https://github.com/bcgov/parks-reso-public/pull/304/commits/1a599013cd01abb93ec6705be0b8577b355eb442